zerver/lib/email_notifications.py

```diff
@@ -117,6 +118,54 @@ def make_emoji_img_elem(emoji_span_elem: CSSSelector) -> Dict[str, Any]:
     content = lxml.html.tostring(fragment).decode('utf-8')
     return content
 
+def fix_spoilers_in_html(content: str, language: str) -> str:
+    with override_language(language):
+        spoiler_title: str = _("Open Zulip to see the spoiler content")
```

Comment:
We're adding yet another string to be translated this close to a release. If there's an elegant way around this, we should try that.